pkg/apis/crd/v1beta1/types.go

```diff
@@ -234,7 +235,7 @@ type SubnetInfo struct {
 	// Prefix length for the subnet, e.g. 24.
 	PrefixLength int32 `json:"prefixLength"`
 	// VLAN ID for this subnet. Default is 0. Valid value is 0~4094.
-	VLAN int32 `json:"vlan,omitempty"`
+	VLAN uint16 `json:"vlan,omitempty"`
```

Comment:
We avoid unsigned integers in the API by convention: https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#primitive-types

More specifically, for integers the first choice should always be int32, with int64 as a possible fallback.

Range constraints should be defined as part of the openAPI spec, not through the primitive type.

Reply:
OK, I will change it back. I updated this one previously because IPPool VLAN used uint16 and I will update that one to int32 to make the two consistent.